src/test/java/dev/stalla/model/ItunesCategoryFactoryTest.java

```diff
@@ -0,0 +1,126 @@
+package dev.stalla.model;
+
+import dev.stalla.model.itunes.ItunesCategory;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class ItunesCategoryFactoryTest {
+
+    @Test
+    @DisplayName("should be able to access the correct iTunes category instances from the companion object")
+    public void testItunesCategoryFactory() {
+        assertEquals(ItunesCategory.from("Arts"), ItunesCategory.Arts);
+        assertEquals(ItunesCategory.from("Books"), ItunesCategory.Books);
```

Comment:
These tests were a bit more relevant in the enum implementation (check if delegation was set up correctly). But they've been there already, and were laborious to write, so let's keep them.

Reply:
Remnant from a previous version to confirm consistent Java API primary. Could be Kotlin now, but just did have the nerve to refactor the entire category list again 😅

Reply:
Sounds like something to get to later on:)